### DISCUSSION TITLE
keycodes for keybinding

### DISCUSSION THREAD

**Discussion by minsoehan** [jarun/nnn#1886] [General]
I have been using nnn for long time. It is my main file manager. Thanks for it.
When I use non us keyboard layout without per-window keyboard layout function in my window manager, it is very inconvenient. Pressing keys are not functioning when I am on non us keyboard layout in nnn. I think it would be great if nnn has keycodes keybinding like `bindsym --to-code` in sway.

**Comment by N-R-K** [comment]
nnn isn't a gui application. It's a terminal one, it doesn't interact directly with the keyboard/keycodes, it uses whatever the terminal sends it.
